custom/icds_reports/utils/aggregation_helpers/distributed/agg_dashboard_acitivity.py

```diff
@@ -44,29 +44,69 @@ def get_dashboard_users(self):
             if dashboard_uname_rx.match(uname)
         }
 
+    @cached_property
+    def transformed_locations(self):
+        """
+        :return: Returns a dict containing location_id as key and its info(loc_level,parents) as value
+                eg: {block_loc_id1: {'loc_level':3, parents:{
```

Comment:
nit: this is pretty wide. might be easier to read if written as
```
{
    block_loc_id1: {
        'loc_level': 3, 
        'parents': {
            'district_id': district_loc_id1,
            'state_id': state_loc_id1
        }
    }
}
```

Reply:
it was resolved.